docs/topics/interfaces.md

```diff
@@ -114,3 +114,23 @@ Now, if you derive a concrete class *C* from *A*, you have to override *bar()* a
 However, if you derive *D* from *A* and *B*, you need to implement all the methods that you have
 inherited from multiple interfaces, and you need to specify how exactly *D* should implement them. This rule applies
 both to methods for which you've inherited a single implementation (*bar()*) and to those for which you've inherited multiple implementations (*foo()*).
+
+## JVM default method generation for interface functions
+
+On the JVM, functions declared in interfaces are compiled to default methods.
+You can control this behavior using the `-jvm-default` compiler option with the following values:
+
+* `enable` (default): generates default implementations in interfaces and includes bridge functions in subclasses and `DefaultImpls` classes. Use this mode to maintain binary compatibility with older Kotlin versions.
```

Comment:
The values in the table are capitalized; should they be capitalized here?

Reply:
Uppercase ones are the values of the Gradle option. Here we're talking about the compiler option, whose values are lowercase